Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/AboutTextManagerTest.java

```diff
@@ -123,4 +127,12 @@ public void testGetAboutTextByProjectId() throws QuadrigaStorageException {
         assertEquals("Test1", result.getId());
     }
 
+    @Test(expected = NullPointerException.class)
+    public void testEmptyProject() throws QuadrigaStorageException {
+
+        Mockito.when(mockedAboutTextDAO.getDTOByProjectId("test24")).thenReturn(null);
+        aboutTextManager.getAboutTextByProjectId("test24");
```

Comment:
why would that throw a nullpointer? Nullpointers should rarely be thrown but appropriately handled. It's not unthinkable that this method might be called with a not existing id and it can't throw a nullpointer.